actionlist/app/containers/ActionListContainer.js

```diff
@@ -1,17 +1,69 @@
-import React from 'react';
-
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import {
   View,
+  DrawerLayoutAndroid,
 } from 'react-native';
 
 import Header from '../components/header';
 import DisplayList from '../components/display_list';
+import SideMenu from '../components/SideMenu';
+import { toggleDrawer } from '../actions/action_items';
+
+class ActionListContainer extends Component {
```

Comment:
1. We are trying to implement the DrawerLayoutAndroid based on the click of menu icon and not the swipe action. While the swipe action is implicitly handled by the native component (which we have disabled) The click needs to be handled this way.
2. There is no DRAWER_OPENED or CLOSED states which can render the component as desired. It needs to be opened only via the openDrawer() function.
3. So we get the response for the click from the reducer and use it with componentDidUpdate and then open or close the drawer with the ref.
4. Since I am triggering the drawer based on the menu button click, Menu Icon button won't be visible to close it when the drawer is in open state. We can close the drawer by pressing somewhere in the screen, this action is captured by DrawerLayoutAndroid.onDrawerClose to set the state variable.